h2o-algos/src/test/java/hex/coxph/CoxPHTest.java

```diff
@@ -335,4 +335,34 @@ private static Frame rebalanceToAllNodes(Frame fr, Key<Frame> rebalancedKey) {
     return fr;
   }
 
+  @Test
+  public void testJavaScoring() {
```

Comment:
Please also add a test case where the MOJO itself is downloaded and imported back into H2O. This could be tested in the `Generic` model tests.

Reply:
This is tested in python but I can add a test for in Generic too

Reply:
Generic is different - The generic algo needs to import the MOJO and do the scoring properly. Which means we need at least Generic tests as well. There is extra logic involved.

Reply:
As you wish ;-)